src/partitioning.c

```diff
@@ -261,13 +261,17 @@ ts_partitioning_func_apply(PartitioningInfo *pinfo, Oid collation, Datum value)
 }
 
 TSDLLEXPORT Datum
-ts_partitioning_func_apply_slot(PartitioningInfo *pinfo, TupleTableSlot *slot, bool *isnull)
+ts_partitioning_func_apply_slot(PartitioningInfo *pinfo, TupleTableSlot *slot, bool *isnull,
+								AttrNumber partition_col_idx)
```

Comment:
Might be good to add a small comment that mentions that this function is applied to chunk tuples also, which might have different column order, and that's why we pass the `partition_col_idx` separately instead of the one in `pinfo`.

Reply:
Added comment.